examples/index.jsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Link, Route, Outlet } from 'react-router-dom';
+import { FormRouter } from '@department-of-veterans-affairs/va-forms-system-core';
+import FormApp from './multi-page';
 
-const App = () => (
-  <div
-    style={{ display: 'flex', alignItems: 'center', flexDirection: 'column' }}
-  >
-    <h1>VA Forms System Core Examples</h1>
-    <ul>
-      <li>
-        <a href="simple-form">Simple form</a>
-      </li>
-      <li>
-        <a href="multipage-form">Multipage form</a>
-      </li>
-      <li>
-        <a href="radio-group">Radio group</a>
-      </li>
-      <li>
-        <a href="form-state">Form State</a>
-      </li>
-    </ul>
-  </div>
-);
+import '@department-of-veterans-affairs/component-library/dist/main.css';
+import { defineCustomElements } from '@department-of-veterans-affairs/component-library';
+void defineCustomElements();
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const initialValues = {
```

Comment:
Used as `initialValues` for formik. Ideally, these would be collected from a json-schema and passed in by our users. Specifying initial values helps maintain form state when navigating to different pages.